Headlines/src/ViewAnimator/AnimationType.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  AnimationType.swift
+//  Pods-ViewAnimator_Example
+//
+//  Created by Marcos Griselli on 10/5/17.
+//
+
+import UIKit
+
+/// AnimationType available to perform/
+///
+/// - from: Animation with direction and distance.
+/// - zoom: Zoom animation.
+/// - rotate: Rotation animation.
+public enum AnimationType: Animation {
+
+    case from(direction: Direction, offset: CGFloat)
+    case zoom(scale: CGFloat)
+    case rotate(angle: CGFloat)
+    
+    /// Creates the corresponding CGAffineTransform for AnimationType.from.
+    public var initialTransform: CGAffineTransform {
+        switch self {
+        case .from(let direction, let offset):
+            let sign = direction.sign
+            if direction.isVertical { return CGAffineTransform(translationX: 0, y: offset * sign) }
+            return CGAffineTransform(translationX: offset * sign, y: 0)
+        case .zoom(let scale):
+             return CGAffineTransform(scaleX: scale, y: scale)
+        case .rotate(let angle):
+            return CGAffineTransform(rotationAngle: angle)
+        }
+    }
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a5cc79d8b0bc8bc433e47d6a58e00624ade77840/Headlines/src/ViewAnimator/AnimationType.swift#L34"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AnimationType.swift:34</code></td>
    </tr>
  </tbody>
</table>